protocol/x/clob/keeper/process_operations.go

```diff
@@ -532,6 +532,21 @@ func (k Keeper) PersistMatchLiquidationToState(
 		return err
 	}
 
+	notionalQuoteQuantums, err := k.perpetualsKeeper.GetNetNotional(
+		ctx,
+		matchLiquidation.PerpetualId,
+		new(big.Int).SetUint64(matchLiquidation.TotalSize),
+	)
+	if err != nil {
+		return err
+	}
+
+	telemetry.IncrCounterWithLabels(
+		[]string{types.ModuleName, metrics.LiquidationOrderNotionalQuoteQuantums, metrics.DeliverTx},
+		float32(notionalQuoteQuantums.Uint64()),
```

Comment:
For my understanding wouldn't this overflow for longs?

Reply:
Changing this to take absolute value of notional quote quantums